.github/workflows/Connector Master SDK Workflow.yml

```diff
@@ -82,6 +82,33 @@ jobs:
         run: dotnet test "${{ steps.findSlnFile.outputs.solutionFilePath }}" --filter TestCategory!=IntegrationTest --logger "trx;logfilename=unitTestResults.trx" --collect "XPlat Code Coverage" -- DataCollectionRunSettings.DataCollectors.DataCollector.Configuration.Format=cobertura,opencover
         continue-on-error: true
 
+      - name: Install Connector Validator
+        run: dotnet tool install -g Skyline.DataMiner.CICD.Tools.Validator
+  
+      - name: Run Connector Validator
+        run: dataminer-validator validate-protocol-solution --solution-path "${{ github.workspace }}" --output-directory "${{ github.workspace }}" --output-file-name "validateResults" 
+        continue-on-error: true
+
+      - name: Archive Results
+        uses: actions/upload-artifact@v4
+        with:
+          name: validateResults
+          path: ${{ github.workspace }}/validateResults.json
```

Comment:
by default, the validator will generate the results in a json and HTML file. If you only archive the json result, then it's better to specify only that output format.

Reply:
Adjusted it, so I archive both json & html. The html will be nicer to look at. The Json can be used in other jobs outside of the reusable workflow.